addons/io_hubs_addon/hubs_session.py

```diff
@@ -58,6 +58,22 @@
     try { params["roomName"] = APP?.hub?.name || APP?.hub?.slug || APP?.hub?.hub_id; } catch(e) {};
     return params;
 """
+JS_WAYPOINT_UPDATE = """
+    window.__scene_debugger_scene_update_listener = () => {
+        try {
+            setTimeout(() => {
+                window.location = `${window.location.href}#${arguments[0]}`;
+                const mat = APP.world.scene.getObjectByName("__scene_debugger_viewpoint").matrixWorld;
```

Comment:
I think this should also use `${arguments[0]` instead of hard coding `__scene_debugger_viewpoint`

Reply:
This is from an initial attempt, the current approach doesn't use JS injection and relies on adding an explicit object in the scene.